src/Bicep.Decompiler/TemplateConverter.cs

```diff
@@ -655,7 +710,93 @@ StringSyntax createFakeModulePath(string templateLinkExpression)
             return SyntaxFactory.CreateStringLiteral(filePath);
         }
 
-        private SyntaxBase ProcessCondition(JObject resource, ObjectSyntax body)
+        private (SyntaxBase body, IEnumerable<SyntaxBase> decorators) ProcessResourceCopy(JObject resource, Func<JObject, SyntaxBase> resourceBodyFunc)
+        {
+            if (TemplateHelpers.GetProperty(resource, "copy")?.Value is not JObject copyProperty)
+            {
+                return (resourceBodyFunc(resource), Enumerable.Empty<SyntaxBase>());
+            }
+
+            TemplateHelpers.AssertUnsupportedProperty(resource, "condition", "The 'copy' property is not supported in conjunction with the 'condition' property");
+
+            var name = TemplateHelpers.AssertRequiredProperty(copyProperty, "name", "The copy object is missing a \"name\" property");
+            var count = TemplateHelpers.AssertRequiredProperty(copyProperty, "count", "The copy object is missing a \"count\" property");
+
+            // TODO implement when we have batchSize support (https://github.com/Azure/bicep/issues/1625)
```

Comment:
>TODO [](start = 15, length = 4)

Thanks for leaving a note :)

Reply:
https://www.youtube.com/watch?v=IJw2lFWJ4aY